### ISSUE TITLE
New submissions for Mon, 11 Mar 24

### ISSUE THREAD

**Issue by olozhika** [olozhika/ArXivDaily_StarFormation#236]
# New submissions for Mon, 11 Mar 24
Auto update Star Formation & Molecular Cloud papers at about 2:30am UTC (10:30am Beijing time) every weekday.

阅读 `Usage.md`了解如何使用此repo实现个性化的Arxiv论文推送

See `Usage.md` for instructions on how to personalize the repo. 

Keyword list: ['star formation', 'star-forming', 'molecular cloud', 'interstellar medium', 'cloud', 'clump', 'core', 'filament', 'atomic gas', 'N-PDF']

Excluded: ['galaxies', 'galaxy cluster', ' AGN ']

#### Atmospheric Waves Driving Variability and Cloud Modulation on a  Planetary-Mass Object
 - **Authors:** Michael K. Plummer, Ji Wang, Étienne Artigau, René Doyon, Genaro Suárez
 - **Subjects:** Earth and Planetary Astrophysics (astro-ph.EP); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2403.04840
 - **Pdf link:** https://arxiv.org/pdf/2403.04840
 - **Abstract**
 Planetary-mass objects and brown dwarfs at the transition from relatively red L dwarfs to bluer mid-T dwarfs $\left(\rm{T}_{eff}\sim1300 \ \text{K} \right)$ show enhanced spectrophotometric variability. An open question is whether this variability is caused by atmospheric planetary-scale (Kelvin or Rossby) waves or by large spots associated with the precipitation of silicate and metal clouds. We applied both waves and spotted models to fit near-infrared (NIR), multi-band ($Y$/$J$/$H$/$K$) photometry of SIMP J013656.5+093347 (hereafter SIMP0136), collected at the Canada-France-Hawaii Telescope using the Wide-field InfraRed Camera. SIMP0136 is a planetary-mass object (12.7$\pm1.0 \ \rm{M_J}$) at the L/T transition (T2$\pm0.5$) known to exhibit light curve evolution over multiple rotational periods. We measure the maximum peak-to-peak variability of $6.17\pm0.46\%$, $6.45\pm0.33\%$, $6.51\pm0.42\%$, and $4.33\pm0.38\%$ in the $Y$, $J$, $H$, and $K$ bands respectively, and find evidence that wave models are preferred for all four NIR bands. Furthermore, we determine the spot size necessary to reproduce the observed variations is larger than the Rossby deformation radius and Rhines scale, which is unphysical. Through the correlation between light curves produced by the waves and associated color variability, we find evidence of planetary-scale, wave-induced cloud modulation and breakup, similar to Jupiter's atmosphere and supported by general circulation models. We also detect a $93.8^{\circ}\pm7.4^{\circ}$ ($12.7\sigma$) phase shift between the $H-K$ and $J-H$ color time series, providing evidence for complex vertical cloud structure in SIMP0136's atmosphere.
#### The extreme coronal line emitter AT 2022fpx: Varying optical  polarization properties and late-time X-ray flare
 - **Authors:** Karri I. I. Koljonen, Ioannis Liodakis, Elina Lindfors, Kari Nilsson, Thomas M. Reynolds, Panos Charalampopoulos, Konstantinos Kouroumpatzakis, Callum McCall, Helen E. Jermak, Iain A. Steele, Juan Carbajo-Hijarrubia
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2403.04877
 - **Pdf link:** https://arxiv.org/pdf/2403.04877
 - **Abstract**
 Supermassive black holes have been known to disrupt passing stars, producing outbursts called tidal disruption events (TDEs). TDEs have recently gained attention due to their unique dynamics and emission processes, which are still not fully understood. Especially, the so-called optical TDEs, are of interest as they often exhibit delayed or obscured X-ray emission from the accretion disk, making the origin of the prompt emission unclear. In this paper, we present the optical photo-, spectro-, and polarimetry of a recent TDE candidate AT 2022fpx, in addition to accompanying monitoring observations in ultraviolet and X-rays. The optical spectra of AT 2022fpx show Bowen fluorescence as well as highly-ionized iron emission lines, which are characteristic of extreme coronal line emitters. Additionally, the source exhibits variable but low-polarized emission at the outburst peak, with a clear rotation of the polarization angle. X-ray emission observed later in the decay appear flare-like but is consistent with constant temperature black-body emission. The overall outburst decay is slower than for typical TDEs, and resembles more the ones seen from Bowen fluorescence flares. These observations suggest that AT 2022fpx could be a key source in linking different long-lived TDE scenarios. Its unique characteristics, such as the extreme coronal line emission, variable polarization, and the delayed X-ray flare, can be attributed to the outer shock scenario or a clumpy torus surrounding the supermassive black hole. Further studies, especially in the context of multi-wavelength observations, are crucial to fully understand the dynamics and emission mechanisms of these intriguing astrophysical events.
#### Deep-Learning Classification and Parameter Inference of Rotational  Core-Collapse Supernovae
 - **Authors:** Solange Nunes, Gabriel Escrig, Osvaldo G. Freitas, José A. Font, Tiago Fernandes, Antonio Onofre, Alejandro Torres-Forné
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE); General Relativity and Quantum Cosmology (gr-qc)
 - **Arxiv link:** https://arxiv.org/abs/2403.04938
 - **Pdf link:** https://arxiv.org/pdf/2403.04938
 - **Abstract**
 We test Deep-Learning (DL) techniques for the analysis of rotational core-collapse supernovae (CCSN) gravitational-wave (GW) signals by performing classification and parameter inference of the GW strain amplitude ($D \cdot \Delta h $) and the maximum (peak) frequency $(f_\textrm{peak})$, attained at core bounce. Our datasets are built from a catalog of numerically generated CCSN waveforms assembled by Richers 2017. Those waveforms are injected into noise from the Advanced LIGO and Advanced Virgo detectors corresponding to the O2 and O3a observing runs. For a signal-to-noise ratio (SNR) above 5, our classification network using time series detects Galactic CCSN GW signals buried in detector noise with a false positive rate (FPR) of 0.10% and a 98% accuracy, being able to detect all signals with SNR>10. The inference of $f_\textrm{peak}$ is more accurate than for $D \cdot \Delta h $, particularly for our datasets with the shortest time window (0.25 s) and for a minimum SNR=15. From the calibration plots of predicted versus true values of the two parameters, the standard deviation ($\sigma$) and the slope deviation with respect to the ideal value are computed. We find $\sigma_{D \cdot \Delta h}$ = 52.6cm and $\sigma_{f_\textrm{peak}}$ = 18.3Hz, with respective slope deviations of 11.6% and 8.3%. Our best model is also tested on waveforms from a recent CCSN catalog built by Mitra 2023, different from the one used for the training. For these new waveforms the true values of the two parameters are mostly within the 1$\sigma$ band around the network's predicted values. Our results show that DL techniques hold promise to infer physical parameters of Galactic rotational CCSN even in the presence of real (non-Gaussian) noise conditions from current GW detectors.
#### The Local Bubble is a Local Chimney: A New Model from 3D Dust Mapping
 - **Authors:** Theo J. O'Neill, Catherine Zucker, Alyssa A. Goodman, Gordian Edenhofer
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2403.04961
 - **Pdf link:** https://arxiv.org/pdf/2403.04961
 - **Abstract**
 Leveraging a high-resolution 3D dust map of the solar neighborhood from Edenhofer et al. (2023), we derive a new 3D model for the dust-traced surface of the Local Bubble, the supernova-driven cavity surrounding the Sun. We find that the surface of the Local Bubble is highly irregular in shape, with its peak extinction surface falling at an average distance of 170 pc from the Sun (spanning 70-600+ pc) with a typical thickness of 35 pc and a total dust-traced mass of $(6.0 \pm 0.7) \times 10^5 \ \rm{M}_{\odot}$. The Local Bubble displays an extension in the Galactic Northern hemisphere that is morphologically consistent with representing a "Local Chimney." We argue this chimney was likely created by the "bursting" of this supernova-driven superbubble, leading to the funneling of interstellar medium ejecta into the lower Galactic halo. We find that many well-known dust features and molecular clouds fall on the surface of the Local Bubble and that several tunnels to other adjacent cavities in the interstellar medium may be present. Our new, parsec-resolution view of the Local Bubble may be used to inform future analysis of the evolution of nearby gas and young stars, the investigation of direct links between the solar neighborhood and the Milky Way's lower halo, and numerous other applications.
#### Infrared Emission of Specific Polycyclic Aromatic Hydrocarbon Molecules:  Cyanonaphthalenes
 - **Authors:** Kaijun Li, Aigen Li, Xuejuan Yang, Taotao Fang
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2403.05043
 - **Pdf link:** https://arxiv.org/pdf/2403.05043
 - **Abstract**
 The unidentified infrared emission (UIE) features at 3.3, 6.2, 7.7, 8.6, 11.3 and 12.7 micron are ubiquitously seen in a wide variety of astrophysical regions and commonly attributed to polycyclic aromatic hydrocarbon (PAH) molecules. However, the unambiguous identification of any individual, specific PAH molecules has proven elusive until very recently two isomers of cyanonapthalene, which consists of two fused benzene rings and substitutes a nitrile (-CN) group for a hydrogen atom, were discovered in the Taurus Molecular Cloud based on their rotational transitions at radio frequencies. To facilitate the James Webb Space Telescope (JWST) to search for cyanonapthalenes in astrophysical regions, we model the vibrational excitation of cyanonapthalenes and calculate their infrared emission spectra in a number of representative astrophysical regions. The model emission spectra and intensities will allow JWST to quantitatively determine or place an upper limit on the abundance of cyanonapthalenes.
#### A Multiwavelength Machine-learning Approach to Classifying X-ray Sources  in the Fields of Unidentified 4FGL-DR4 sources
 - **Authors:** Hui Yang, Jeremy Hare, Oleg Kargaltsev
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE)
 - **Arxiv link:** https://arxiv.org/abs/2403.05068
 - **Pdf link:** https://arxiv.org/pdf/2403.05068
 - **Abstract**
 A large fraction of Fermi-LAT sources in the 4th Fermi LAT 14-year Catalog (4FGL) still remain unidentified (unIDed). We continued to improve our machine learning pipeline (MUWCLASS) and used it to classify 1206 X-ray sources located within the extent of 73 unIDed 4FGL sources with Chandra X-ray Observatory (CXO) observations included in the Chandra Source Catalog 2.0. Recent improvements to our pipeline include astrometric corrections, probabilistic cross-matching to lower-frequency counterparts, and a more realistic oversampling method. X-ray sources are classified into 8 broad pre-determined astrophysical classes defined in the updated training dataset which we also release. The classifications give 115 plausible X-ray counterparts to 45 GeV sources. We present details of the machine learning classification, describe the pipeline improvements, and perform additional spectral and variability analysis for brighter sources. We identify 3 GeV sources as isolated neutron star candidates, 1 as an X-ray binary candidate, 16 as active galactic nucleus candidates, 7 sources are associated with star-forming regions, and 9 are ambiguous cases. For the remaining 37 unIDed 4FGL sources, we could not identify any plausible counterpart in X-rays or they are too close to the Galactic Center. Finally, we outline the observational strategies and further improvements in the pipeline that can lead to more accurate classifications.
#### Proper motion of Cygnus loop shock filaments
 - **Authors:** M. Vucetic, N. Milanovic, D. Urosevis, J. Raymond, D. Onic, S. Milosevic, N. Petrov
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE); Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2403.05215
 - **Pdf link:** https://arxiv.org/pdf/2403.05215
 - **Abstract**
 We determined shock speed in the Galactic supernova remnant Cygnus Loop, using proper motion of its optical filaments and the latest estimate for its distance. The proper motion was measured by comparing H$\alpha$ images of the remnant observed in two epochs: 1993 (Kitt Peak National Observatory) and 2018/2019 (National Astronomical Observatory Rozhen and Astronomical station Vidojevica). We derived shock speed for 35 locations along different filaments, which is twice as much as in earlier studies of north-eastern part of Cygnus Loop. For the first time, we have measured shock speed of radiative filaments in this region. Three of the analyzed locations where we measured proper motion of filaments are radiative, based on their presence in [SII] images from the second epoch. The other filaments are non-radiative. The speed we obtained for the non-radiative filaments is in the range of 240-650 $\mathrm{km\ s^{-1}}$, with an estimate for the uncertainty of 70 $\mathrm{km\ s^{-1}}$. These values are mostly in agreement with previous studies. The radiative filaments have lower speed of 100-160$\pm${70} $\mathrm{km\ s^{-1}}$, which is in agreement with the assumption that they are older in evolutionary terms. This clear distinction between the speed of the two types of filaments proves that the [SII] emission can be used for identifying radiative filaments in supernova remnants.
#### Complex organic molecules uncover deeply embedded precursors of hot  cores
 - **Authors:** Laure Bouscasse, Timea Csengeri, Friedrich Wyrowski, K. M. Menten, S. Bontemps
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2403.05237
 - **Pdf link:** https://arxiv.org/pdf/2403.05237
 - **Abstract**
 During the process of star formation, the dense gas undergoes significant chemical evolution leading to the emergence of a rich variety of molecules associated with hot cores and hot corinos. However, the physical and chemical conditions involved in this evolution are poorly constrained. We provide here a full inventory of the emission from complex organic molecules (COMs) to investigate the physical structure and chemical composition of six high-mass protostellar envelopes. We aim to investigate the conditions for the emergence of COMs in hot cores. We performed an unbiased spectral survey towards six infrared-quiet massive clumps between 159 GHz and 374 GHz with the APEX 12 m telescope. We detect up to 11 COMs, of which at least five COMs are detected towards all sources. Towards all the objects, most of the COM emission is found to be cold, with respect to the typical temperatures at which COMs are found, with a temperature of 30 K and extended with a size of ~0.3 pc. Although for our sample of young massive clumps the bulk of the gas has a cold temperature, we also detect emission from COMs originating from the immediate vicinity of the protostar revealing a compact and hot component of the envelope. Only three out of the six sources exhibit a hot gas component. We find a gradual emergence of the warm component in terms of size and temperature, together with an increasing molecular complexity, allowing us to establish an evolutionary sequence for our sample based on COMs. Our findings confirm that our sample of infrared-quiet massive clumps are in an early evolutionary stage during which the bulk of the gas is cold. The presence of COMs is found to be characteristic of these early evolutionary stages. We suggest that the emergence of hot cores is preceded by a phase in which mostly O-bearing COMs appear first with similar abundances to hot corinos albeit with larger source sizes.
#### An implicit algorithm for simulating the dynamics of small dust grains  with smoothed particle hydrodynamics
 - **Authors:** Daniel Elsender, Matthew R. Bate
 - **Subjects:** Earth and Planetary Astrophysics (astro-ph.EP); Instrumentation and Methods for Astrophysics (astro-ph.IM)
 - **Arxiv link:** https://arxiv.org/abs/2403.05345
 - **Pdf link:** https://arxiv.org/pdf/2403.05345
 - **Abstract**
 We present an implicit method for solving the diffusion equation for the evolution of the dust fraction in the terminal velocity approximation using dust-as-mixture smoothed particle hydrodynamics (SPH). The numerical scheme involves casting the dust diffusion equation into implicit form, rearranging into its resolvent cubic equation and solving analytically. This method is relevant for small grains that are tightly coupled to the gas, such as sub-micron dust grains in the interstellar medium or millimetre-sized dust grains in protoplanetary discs. The method avoids problems with the variable used to evolve the dust fraction becoming negative when evolved explicitly and is fast and accurate, avoiding the need for dust stopping time limiters and significantly reducing computational expense. Whilst this method is an improvement over using the explicit terminal velocity approximation method, as with any dust-as-mixture method it still fails to give accurate solutions in the limit of large (weakly coupled) grains.
#### Radiation transport methods in star formation simulations
 - **Authors:** Richard Wünsch
 - **Subjects:** Instrumentation and Methods for Astrophysics (astro-ph.IM)
 - **Arxiv link:** https://arxiv.org/abs/2403.05410
 - **Pdf link:** https://arxiv.org/pdf/2403.05410
 - **Abstract**
 Radiation transport plays a crucial role in star formation models, as certain questions within this field cannot be accurately addressed without taking it into account. Given the high complexity of the interstellar medium from which stars form, numerical simulations are frequently employed to model the star formation process. This study reviews recent methods for incorporating radiation transport into star formation simulations, discussing them in terms of the used algorithms, treatment of radiation frequency dependence, the interaction of radiation with the gas, and the parallelization of methods for deployment on supercomputers. Broadly, the algorithms fall into two categories: (i) moment-based methods, encompassing the flux-limited diffusion approximation, M1 closure, and variable Eddington tensor methods, and (ii) methods directly solving the radiation transport equation, including forward and reverse ray tracing, characteristics-based methods, and Monte Carlo techniques. Beyond discussing advantages and disadvantages of these methods, the review also lists recent radiation hydrodynamic codes implemented the described methods.

by olozhika (Xing Yuchen). 

2024-03-11
